lib/demo_web/router.ex

```diff
@@ -23,4 +26,37 @@ defmodule DemoWeb.Router do
   # scope "/api", DemoWeb do
   #   pipe_through :api
   # end
+
+  ## New routes
```

Comment:
```suggestion
  ## Authentication routes
```
I like the idea of having a comment that indicates these are the new authentication routes. Do you think the generators should leave this comment in as `## Authentication routes`?

Reply:
I like this, yes.